build/dist

```diff
@@ -257,9 +253,8 @@ cp "$KYUUBI_HOME/externals/kyuubi-flink-sql-engine/target/kyuubi-flink-sql-engin
 cp "$KYUUBI_HOME/externals/kyuubi-spark-sql-engine/target/kyuubi-spark-sql-engine_${SCALA_VERSION}-${VERSION}.jar" "$DISTDIR/externals/engines/spark"
 
 # Copy trino engines
-cp -r "$KYUUBI_HOME/externals/kyuubi-trino-engine/bin/" "$DISTDIR/externals/engines/trino/bin/"
-chmod a+x "$DISTDIR/externals/engines/trino/bin/trino-engine.sh"
-cp "$KYUUBI_HOME/externals/kyuubi-trino-engine/target/kyuubi-trino-engine_${SCALA_VERSION}-${VERSION}.jar" "$DISTDIR/externals/engines/trino/jars"
+cp "$KYUUBI_HOME/externals/kyuubi-trino-engine/target/kyuubi-trino-engine_${SCALA_VERSION}-${VERSION}.jar" "$DISTDIR/externals/engines/trino"
+cp -r "$KYUUBI_HOME/externals/kyuubi-trino-engine/target/scala-$SCALA_VERSION/jars/" "$DISTDIR/externals/engines/trino"
```

Comment:
The command will put those jars under `engines/trino` directly.  Shell we make them put `trino/jars` or `trino/scala-$SCALA_VERSION/jars/`

Reply:
I don't agree, it make huge gaps between engines

Reply:
what's more, most of the dependencies here
- duplicate with kyuubi servers, making our release tar oversize
- uncurated, we don't carefully manage the transitive jars, the behavior just inherits from the parent's maven-dependency-jar, which is dangerous for making an apache-approved release.

Reply:
Get it.